### ISSUE TITLE
Conditions in v10.6.2

### ISSUE THREAD

**Issue by terafirmanz** [directus/directus#19757]
### Describe the Bug

After upgrading to v10.6.2 from 10.6.1 all conditions on fields no longer show in the UI and any new ones don't function.

### To Reproduce

Upgrade to v10.6.2 and create a new condition that hides a field. the rule will not apply when triggered.

Also go to a field that does have a condition and edit it. Under conditions, none will be shown.

Downgrade to 10.6.1 and all existing conditions show again but any created under version 10.6.2 now show with this error:
![image](https://github.com/directus/directus/assets/53923271/2d0f5871-e0bb-455a-b614-202ab09ae089)

### Directus Version

v10.6.2

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by terafirmanz** [comment]
FYI the field that error is shown for has nothing to do with a repeater other than that there is a repeater on in the same collection.

**Comment by DanielBiegler** [comment]
Potentially relevant: #19571

**Comment by Pachat** [comment]
Tried on detail group and confirm the bug.

**Comment by paescuj** [comment]
Hey there 👋 

The issue has been fixed in #19799.

If you've added or edited Field Conditions **under [v10.6.2](https://github.com/directus/directus/releases/tag/v10.6.2)** you need to manually adjust the value. Note that no other versions are affected by this.

You are affected if the following message appears in the "Conditions" tab under the Field Settings:

<img width="606" alt="Screenshot 2023-09-25 at 18 56 08" src="https://github.com/directus/directus/assets/5363448/f50e3484-e821-45a4-92a9-93a1d6d36cd4">

In this case, for the affected conditions, open the Field Menu (1) and click on "Edit Raw Value" (2):

<img width="621" alt="Screenshot 2023-09-25 at 19 00 15" src="https://github.com/directus/directus/assets/5363448/be9ecfb8-6390-4dd7-a78b-275925648f42">

Now, in the Raw Editor remove the following part **at the beginning** (up to but not including the `[` character)

```
{
    "conditions":
```

and the following character **at the end**

```
}
```

then click on "Done". (See also the full example below)

After that the conditions work again as usual!

### Example

#### Before

```json
{
    "conditions": [
        {
            "name": "my-condition",
            "rule": {
                "_and": [
                    {
                        "example_field": {
                            "_contains": "example"
                        }
                    }
                ]
            },
            "readonly": true,
            "options": {}
        }
    ]
}
```

#### After

```json
[
    {
        "name": "my-condition",
        "rule": {
            "_and": [
                {
                    "example_field": {
                        "_contains": "example"
                    }
                }
            ]
        },
        "readonly": true,
        "options": {}
    }
]
```
